Assets/Code/Sprite/AnimatedSprite/Character/Character.cs

```diff
@@ -16,31 +21,10 @@ public GridCoordinates Position
         set 
         {
             position = value;
+            transform.position = grid.GetTileVector3(value);
```

Comment:
Grumble grumble, movement animation bug, grumble grumble.

P.S. I agree with the change, but I just wanted to leave a comment on the PR showing what the cause of the movement animation bug was. By putting the transform.position assignment here, we removed the UpdateSprite() function. The removal of this function invalidated chained together animations that were just merge in as part of #14 .

Reply:
Bug ticket covering this generated here: https://github.com/darin-ellis7/GridGame/issues/17